tests_async/system/requests/test_download.py

```diff
@@ -291,29 +329,28 @@ async def test_download_to_stream(self, add_files, authorized_transport):
             # Create the actual download object.
             media_url = utils.DOWNLOAD_URL_TEMPLATE.format(blob_name=blob_name)
             stream = io.BytesIO()
-      
+
             download = self._make_one(media_url, stream=stream)
             # Consume the resource.
             response = await download.consume(authorized_transport)
             assert response.status == http_client.OK
-          
-            #breakpoint()           
-            
-            aiohttp session is closing itself 
+
+            #breakpoint()
```

Comment:
can w remove this? Also this whole bit still seems commented out. we should track that the download tests fail, likely better to mark as skipped also.

Reply:
Added a pytest skip marker - the test works differently now since every response comes back with body in a stream, so therefore i presume the test would either need to be reworked from the sync version or deleted